src/scenes/Order/components/ShippingMethod/ShippingMethod.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import {Grid} from '@material-ui/core'
+
+import styles from './styles.module.css'
+
+class ShippingMethod extends React.Component{
+
+  state={
+    error: ""
+  }
+
+  handleChange = (event) => {
+    this.props.onChange(
+      event.target.value,
+      event.target.id
+    );
+  }
+
+  handleSubmit = (event) => {
+    let check = (
+    document.getElementById('UPS Ground').checked || 
+    document.getElementById('UPS 3 Day Select').checked || 
+    document.getElementById('UPS 2nd Day Air').checked || 
+    document.getElementById('UPS Next Day Air').checked) 
```

Comment:
Totally not correct way of doing things in react. You either pass it as props, or use ref